mteb/tasks/PairClassification/multilingual/RTE3.py

```diff
@@ -0,0 +1,88 @@
+from __future__ import annotations
+
+import datasets
+
+from mteb.abstasks import MultilingualTask
+from mteb.abstasks.AbsTaskPairClassification import AbsTaskPairClassification
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+_LANGS = {
+    "de": ["deu-Latn"],
+    "en": ["eng-Latn"],
+    "fr": ["fra-Latn"],
+    "it": ["ita-Latn"],
+}
+
+
+class RTE3(MultilingualTask, AbsTaskPairClassification):
+    metadata = TaskMetadata(
+        name="RTE3",
+        dataset={
+            "path": "maximoss/rte3-multi",
+            "revision": "d94f96ca5a6798e20f5a77e566f7a288dc6138d7",
+        },
+        description="Recognising Textual Entailment Challenge (RTE-3) aim to provide the NLP community with a benchmark to test progress in recognizing textual entailment",
+        reference="https://aclanthology.org/W07-1401/",
+        category="s2s",
+        type="PairClassification",
+        eval_splits=["test", "validation"],
+        eval_langs=_LANGS,
+        main_score="ap",
+        date=("2023-03-25", "2024-04-15"),
+        form=["written"],
+        domains=None,
+        task_subtypes=["Claim verification"],
```

Comment:
```suggestion
        task_subtypes=["Claim verification", "Textual Entailment"]
```

Reply:
Oh I didn't find "Textual Entailment" in `TASK_SUBTYPE`, is it ok to set this?

Reply:
I can also add "Textual Entailment" to TASK_SUBTYPE (https://github.com/embeddings-benchmark/mteb/blob/main/mteb/abstasks/TaskMetadata.py#L17). In that case I'll also remove "Claim verification"

Reply:
Added Textual Entailment to the Literal TASK_SUBTYPE and used it as subtype.